lms/templates/courseware/courseware.html

```diff
@@ -206,6 +206,16 @@
                     current_score=int(entrance_exam_current_score * 100)
                 )}
             </p>
+            <script type="text/javascript">
+            $(document).ajaxSuccess(function(event, xhr, settings) {
+                if (settings.url.indexOf("xmodule_handler/problem_check") > -1) {
+                    var data = JSON.parse(xhr.responseText);
+                    if (data.entrance_exam_passed){
+                        location.reload();
```

Comment:
actually i am talking about the score updating in string `To access course materials, you must score 50% or higher on this exam. Your current score is 0%. `